src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java

```diff
@@ -18,9 +10,20 @@
 import com.wikia.webdriver.pageobjectsfactory.componentobject.minieditor.MiniEditorPreviewComponentObject;
 import com.wikia.webdriver.pageobjectsfactory.componentobject.photo.PhotoAddComponentObject;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
 
 public class MessageWall extends WikiBasePageObject {
 
+  private static final String newMessageMenu =
+          ".comments li.SpeechBubble.message.message-main:nth-child(1) .buttons";
+  private static final String firstMessageMenu = ".comments li:nth-child(1) .buttons ";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00115)